pkg/errs/errno.go

```diff
@@ -87,6 +87,7 @@ var (
 	ErrCacheOverflow                    = errors.Normalize("cache overflow", errors.RFCCodeText("PD:scheduler:ErrCacheOverflow"))
 	ErrInternalGrowth                   = errors.Normalize("unknown interval growth type error", errors.RFCCodeText("PD:scheduler:ErrInternalGrowth"))
 	ErrSchedulerCreateFuncNotRegistered = errors.Normalize("create func of %v is not registered", errors.RFCCodeText("PD:scheduler:ErrSchedulerCreateFuncNotRegistered"))
+	ErrSchedulerConfigNoImplement       = errors.Normalize("schedulers config no implement", errors.RFCCodeText("PD:scheduler:ErrSchedulerConfigNoImplement"))
```

Comment:
Where do we use it?

Reply:
Can remove it. In the previous impl, `schedulerConfigHandler.GetSchedulerConfig` has an `errNoImplement` which is defined in `/server/api/util.go`. But `GetSchedulerConfigHandler` returns nil only if `RaftCluster` is nil, so this error output is incorrect. So I don't use ErrSchedulerConfigNoImplement.